runtime.md

```diff
@@ -103,6 +103,7 @@ This operation MUST generate an error if it is not provided the container ID.
 Attempting to start a container that does not exist MUST generate an error.
 Attempting to start an already started container MUST have no effect on the container and MUST generate an error.
 This operation MUST run the user-specified code as specified by [`process`](config.md#process-configuration).
+If `process.args` was not configured, the runtime MUST generate an error.
```

Comment:
> Otherwise we need some sort of default.. description.

With optional validation, runtimes that choose not to validate in `start` will have this sort of miss-configuration result in the same sort of [error-workflow that applies to `execve` errors][1].  I don't see a need for a default, and am not sure what sort of default you'd pick in any case.

[1]: https://github.com/opencontainers/runtime-spec/pull/620#discussion_r95419303